zipline/pipeline/pipeline.py

```diff
@@ -79,6 +87,12 @@ def add(self, term, name, overwrite=False):
             else:
                 raise KeyError("Column '{}' already exists.".format(name))
 
+        if not isinstance(term, ComputableTerm):
+            raise TypeError(
+                "{term} is not a valid pipeline column. Did you mean to "
+                "append '.latest'?".format(term=term)
+            )
```

Comment:
Added the column name here but not above. In this case you don't need the name as you're only adding a single term.